README.md

```diff
@@ -57,9 +68,45 @@ imagelab.find_issues(issue_types=issue_types)
 imagelab.report(issue_types=issue_types)
 ```
 
+3. Run CleanVision on a Hugging Face dataset
+```python
+from datasets import load_dataset, concatenate_datasets
+
+# Download and concatenate different splits
```

Comment:
I think it's better to use markdown text to explain this instead of code comment. The main purpose of a notebook is so you can write explanatory text and this notebook hardl has any.

Reply:
Should explain why you are concatenating train & test, eg:

> Since we're interested in generally understanding what issues plague our data, we merge the training and test sets into one larger dataset before running CleanVision. You could alternatively just run the package on these two sets of data separately to obtain two different reports.